### ISSUE TITLE
PRIMARY MODEL: Test various hyperparameters with the YOLOv8 model

### ISSUE THREAD

**Issue by uzFer** [uzFer/APS360-Project#8]
Test and try out hyperparameters for our model. Some are included by the YOLOv8 model: 

- learning rate
- number of epochs
- dropout
- batch size
- patience
- momentum
- optimizer
